docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: /dev/getting-started/paper-plugins
+---
+
+# Paper Plugins
+
+Paper plugins allow developers to take advantage of more modern concepts introduced by Mojang, such as datapacks, to 
+expand the field of what the Paper API is able to introduce.
+
+:::info Experimental
+
+This is experimental and may be subject to change.
+
+:::
+
+- [Bootstrapper](#bootstrapper)
+- [Loader](#loaders)
+- [Differences](#differences)
+
+## How do I use them?
+Similarly to Bukkit plugins, you have to introduce a ``paper-plugin.yml`` file into your jar resources folder.
+Here is an example configuration.
+```yml
+name: Paper-Test-Plugin
+version: ${version}
+main: io.papermc.testplugin.TestPlugin
+description: Paper Test Plugin
+author: PaperMC
+api-version: ${apiversion}
+load: STARTUP
+bootstrapper: io.papermc.testplugin.TestPluginBootstrap
+loader: io.papermc.testplugin.TestPluginLoader
+defaultPerm: FALSE
+permissions:
+dependencies:
+```
+
+Dependency declaration is specified a bit differently than Bukkit plugins, as you are
+able to define if a dependency is required during bootstrapping. This means that any bootstrapping
+logic for your dependencies will be run before yours is run. 
+```yml
+dependencies:
+  - name: DependencyName
+    required: true
+    bootstrap: true
+```
+
+This dependency is ``required``, meaning that if this dependency is not found your plugin will fail to load.
```

Comment:
Maybe a `soft-depend` and `depend` analogy could be used here?

> If a dependency is ``required`` it must be installed on the server or your plugin will fail to load, similar to a plugin that is listed as a ``depend`` in a Bukkit plugin.yml. Otherwise, your plugin will be loaded fine, in the same fashion as a plugin that is a ``soft-depend`` in a Bukkit plugin.yml

Needs a little bit more love but that's the idea